core/src/main/java/com/scalar/db/transaction/common/AbstractDistributedTransaction.java

```diff
@@ -0,0 +1,68 @@
+package com.scalar.db.transaction.common;
```

Comment:
I also added `AbstractDistributedTransaction`, `AbstractDistributedTransactionManager`, `AbstractTwoPhaseCommitTransaction`, and `AbstractTwoPhaseCommitTransactionManager`.